infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/CorfuNettyServerChannel.java

```diff
@@ -1,117 +1,117 @@
-            Map<Pair<UuidMsg, Long>, ChannelHandlerContext> contexts =
-                    message.getPayload().getPayloadCase() == RequestPayloadMsg.PayloadCase.LR_ENTRY ?
-                    contextMapLogEntries : contextMap;
-            contexts.put(Pair.of(message.getHeader().getClusterId(),
-                message.getHeader().getRequestId()),
-                ctx);
-
-            // Send to the adapter for further processing.
-            adapter.receive(message);
-        } catch (Exception e) {
-            log.error("Exception during read!", e);
-        }
-    }
-
-    /**
-     * Channel event that is triggered when a new connected channel is created.
-     *
-     * @param ctx channel handler context
-     * @throws Exception
-     */
-    @Override
-    public void channelActive(final ChannelHandlerContext ctx) {
-        log.info("channelActive: Incoming connection established from: {} Start Read Timeout.",
-                ctx.channel().remoteAddress());
-    }
-
-    /**
-     * Send a netty message through this channel.
-     *
-     * @param outMsg Outgoing message.
-     */
-    public synchronized void sendResponse(ResponseMsg outMsg) {
-        ChannelHandlerContext ctx = getContext(outMsg);
-        if (ctx != null) {
-            ctx.writeAndFlush(outMsg, ctx.voidPromise());
-            log.trace("Sent response: {}", outMsg);
-        } else {
-            log.warn("Netty context not found for request id={}. Dropping message type={}",
-                    outMsg.getHeader().getRequestId(),
-                    outMsg.getPayload().getPayloadCase());
-        }
-    }
-
-    private ChannelHandlerContext getContext(ResponseMsg message) {
-
-        ChannelHandlerContext context;
-
-        if (message.getPayload().getPayloadCase() == ResponsePayloadMsg.PayloadCase.LR_ENTRY_ACK) {
-            // Because ACKs are aggregated (summarized) for a batch of messages, remove all
-            // contexts from this remote cluster which are lower than this request ID (to prevent memory leak, as those
-            // other messages, will never be served)
-            context =
-                contextMapLogEntries.remove(Pair.of(message.getHeader().getClusterId(),
-                    message.getHeader().getRequestId()));
-            contextMapLogEntries.keySet().removeIf(id ->
-                id.getRight() <= message.getHeader().getRequestId() &&
-                Objects.equals(id.getLeft(), message.getHeader().getClusterId()));
-        } else {
-            context =
-                contextMap.remove(Pair.of(message.getHeader().getClusterId(),
-                    message.getHeader().getRequestId()));
-        }
-
-        return context;
-    }
-
-    @Override
-    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
-        log.error("Error in handling inbound message.", cause);
-        ctx.close();
-    }
-}
+//package org.corfudb.infrastructure.logreplication.transport.sample;
```

Comment:
why is everything commented out ?

Reply:
Netty is commented out for now because we are focusing on GRPC currently. And continuing to having netty was polluting the code coverage data. I think that data is a nice point to ensure that our change is not decreasing the code coverage.